adhocracy4/images/mixins.py

```diff
@@ -0,0 +1,61 @@
+from django import forms
+from django.utils.functional import lazy
+from django.utils.translation import gettext_lazy as _
+
+
+# FIXME: backportet from django 1.11. remove this after update
+def _format_lazy(format_string, *args, **kwargs):
+    return format_string.format(*args, **kwargs)
+
+
+format_lazy = lazy(_format_lazy, str)
+
+
+class ImageMetadataMixin(forms.ModelForm):
+    def __init__(self, *args, image_name="image", **kwargs):
+        self.image_name = image_name
+        super().__init__(*args, **kwargs)
+        if self.instance.image:
+            self.initial["image_copyright"] = True
+            self.initial["image_alt_text"] = True
+
+    image_copyright = forms.CharField(
+        required=False,
+        label=format_lazy(_("{image_name} copyright"), image_name="image"),
+        max_length=120,
+        help_text=format_lazy(
+            _("Copyright shown in the {image_name}."), image_name="image"
+        ),
+    )
+
+    image_alt_text = forms.CharField(
+        required=False,
+        label=format_lazy(_("{image_name} alt text"), image_name="image"),
+        max_length=80,
+        help_text=format_lazy(
+            _(
+                "An alternate text serves as a textual description "
+                "of the image content and is read out by screen "
+                "readers. Describe the image in max. 80 characters. "
+                "Example: A busy square with people in summer."
+            ),
+            image_name="image",
+        ),
```

Comment:
not sure if this is valid, i just kinda mushed it in and the image_name is just a stan-in as wanted to get it kind of running so was just putting thing in so it would start